components/list/image/src/index.scss

```diff
@@ -0,0 +1,63 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+@import '~@schibstedspain/sui-image-lazy-load/lib/index';
+
+$m-h-list-image-item-half-gutter: ceil($m-h-list-image-item-full-gutter / 2) !default;
+$m-v-list-image-item-half-gutter: ceil($m-v-list-image-item-full-gutter / 2) !default;
+$m-h-list-image-item-breakpoint-l-gutter: $m-h-list-image-item-full-gutter - 1px !default;
+
+.sui-ListImage {
+  @include media-breakpoint-only(xl) {
+    margin: -($m-v-list-image-item-half-gutter);
+  }
+  @include media-breakpoint-only(l) {
+    margin: (-($m-v-list-image-item-half-gutter)) (-($m-h-list-image-item-breakpoint-l-gutter));
+  }
+  @include media-breakpoint-down(m) {
+    justify-content: space-between;
+  }
+  display: flex;
```

Comment:
you could use the list mixin https://github.com/SUI-Components/theme-basic/blob/master/src/utils/_list.scss

Reply:
Ok. Included sui-list mixin and removed dsiplay and padding style properties in next commit.